docs/gpu_memory_allocation.rst

```diff
@@ -60,3 +60,9 @@ Common causes of OOM failures
 **Running JAX on the display GPU.**
   Use :code:`XLA_PYTHON_CLIENT_MEM_FRACTION` or
   :code:`XLA_PYTHON_CLIENT_PREALLOCATE`.
+
+**Disabling rematerialization HLO pass**
+  Sometimes disabling the rematerialization HLO pass is favorable to avoid 
+  poor remat choices by the compiler. The pass can be disabled by adding
+  :code:`jax.config.update('enable_remat_opt_pass', False)`. But this can 
+  sometimes lead to OOM failures. 
```

Comment:
As we are going to change the default I would write it differently.
Enabling or disabling the automatic remat pass allows different tread-off between computation and memory usage. USAGE.
Note, the algorythm is basic, you can get better results if you do it manually with the jax.remat() API (URL).

Or something like that.